regression/cbmc-incr-smt2/pointers-relational-operators/pointers_assume.c

```diff
@@ -0,0 +1,16 @@
+int main()
+{
+  int *x;
+  int *y;
+  __CPROVER_assume(x > y);
```

Comment:
Please revise these tests -- relations on nondeterministic pointers have no meaning. You need to make those pointers point to the same object.

Reply:
Yes, that's fine. It may confuse one or the other reader that you are using `x` both as a source of nondeterminism and as "target" for the pointers.